test/PrizeDistributionHistory.test.ts

```diff
@@ -211,10 +211,10 @@ describe('PrizeDistributionHistory', () => {
         });
 
         it('should fail to create a new draw when called from non-draw-manager', async () => {
-            const drawPrizeWallet2 = prizeDistributionHistory.connect(wallet2);
+            const prizeDistributorWallet2 = prizeDistributionHistory.connect(wallet2);
```

Comment:
These changes will conflict with the following PR: https://github.com/pooltogether/v4-core/pull/205
So need to make sure we do rename in `PrizeDistributionBuffer.test.ts`.